.coveragerc

```diff
@@ -0,0 +1,17 @@
+[run]
```

Comment:
@k I don't think coverage is setup properly on eng-ops, or rather, I don't see that coverage script running in github as it does in main app.  Is that expected?